File: 1768-merge-strings-alternately/1768-merge-strings-alternately.ts
function mergeAlternately(word1: string, word2: string): string {
    let result = []
    for (let i = 0; i < Math.max(word1.length, word2.length); i++) {
        i >= word1.length ? result.push('') : result.push(word1[i])
        i >= word2.length ? result.push('') : result.push(word2[i])
    }
    return result.join('')
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        minLen = min(len(word1), len(word2))
        i = 0
        res = ''
        while i < minLen:
            res += word1[i] + word2[i]
            i += 1
        if len(word1) > len(word2):
            res += word1[i:]
        elif len(word1) < len(word2):
            res += word2[i:]
        return res